Makefile

```diff
@@ -1,6 +1,6 @@
 CC = /usr/bin/gcc
-CFLAGS = -Wall -g -O3 -Wextra -Wpedantic
-LDLIBS = -lcrypto
+CFLAGS = -Wall -g -O3 -Wextra -Wpedantic #-fsanitize-address-use-after-return=always -fsanitize=address
```

Comment:
Since this is commented out, you should probably revert that change.

Reply:
Ah, it's for security purpose during current development. (and probably in the future as well).